WooCommerce/WooCommerceScreenshots/WooCommerceScreenshots.swift

```diff
@@ -0,0 +1,50 @@
+import XCTest
+
+class WooCommerceScreenshots: XCTestCase {
+
+    override func setUp() {
+
+        continueAfterFailure = false
+
+        if UIDevice.current.userInterfaceIdiom == .pad {
+            XCUIDevice.shared.orientation = UIDeviceOrientation.landscapeLeft
+        } else {
+            XCUIDevice.shared.orientation = UIDeviceOrientation.portrait
+        }
+    }
+
+    func testScreenshots() {
+        // UI tests must launch the application that they test.
+        let app = XCUIApplication()
+        setupSnapshot(app)
+
+        app.launchArguments = ["logout-at-launch", "disable-animations"]
+        app.launch()
+
+        WelcomeScreen()
+            .selectLogin()
+            .proceedWith(email: "")
+            .proceedWithPassword()
+            .proceedWith(password: "")
```

Comment:
Totally doable – I figured this would be replaced by mocks soon, which was why I left it like this.

But super easy to add – done in 98462e3e4484bfe2fcf4df5c72c3444dc65f8846!